XCUITests/BrowsingPDFTests.swift

```diff
@@ -16,8 +16,13 @@ class BrowsingPDFTests: BaseTestCase {
         let element = app/*@START_MENU_TOKEN@*/.webViews/*[[".otherElements[\"Web content\"].webViews",".otherElements[\"contentView\"].webViews",".webViews"],[[[-1,2],[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.children(matching: .other).element.children(matching: .other).element(boundBy: 0)
         element.swipeUp()
         waitforExistence(app.staticTexts["2 of 5"])
-        element.swipeDown()
+
+        repeat {
+            element.swipeDown()
+        } while app.staticTexts["1 of 5"].exists == false
```

Comment:
in case when the text is not rendered, this repeat call will not exit, right? So maybe we should put a limit on how many times it swipes, (5 ~10 should be enough?)

Reply:
@npark-mozilla , You were right, sorry did not take that into account. I made a fix for that, please let me know your thinking about it. Thanks!